config/validator/Application/Regulation/Command/SaveRegulationLocationCommand.xml

```diff
@@ -25,6 +25,31 @@
                 </option>
             </constraint>
         </property>
+        <property name="isEntireStreet">
+            <constraint name="Type">
+                <option name="type">boolean</option>
+            </constraint>
+            <constraint name="When">
+                <option name="expression">!value</option>
```

Comment:
value fait référence à quoi ?

Reply:
C'est la valeur du champ qui est en train d'être validé : https://symfony.com/doc/current/reference/constraints/When.html#expression

> value
    The value of the property being validated (only available when the constraint is applied to a property).